Example/Controllers/SwiftUI/Helper/MiniAppViewMessageDelegator.swift

```diff
@@ -183,6 +183,16 @@ class MiniAppViewMessageDelegator: NSObject, MiniAppMessageDelegate {
     func closeMiniApp(withConfirmation: Bool, completionHandler: @escaping (Result<Bool, MiniAppJavaScriptError>) -> Void) {
         onShoudCloseMiniApp?(withConfirmation)
     }
+
+    func getHostAppThemeColors(completionHandler: @escaping (Result<HostAppThemeColors?, MASDKError>) -> Void) {
+        completionHandler(
+            .success(
+                HostAppThemeColors(
+                    primaryColor: "#FF008C",
+                    secondaryColor: "#7D64BE")
+            )
+        )
+    }
```

Comment:
We shouldn't be doing any Hardcoding for this scenario. We should fetch the values from Settings screen may be. Will create a ticket on the same.

Reply:
We will change this implementation with the [MINI-6107](https://jira.rakuten-it.com/jira/browse/MINI-6107) Jira ticket for it.